x-pack/plugin/esql/src/test/java/org/elasticsearch/xpack/esql/expression/function/AbstractFunctionTestCase.java

```diff
@@ -1149,6 +1151,7 @@ private static void renderKibanaFunctionDefinition(
             builder.endArray();
         }
         builder.field("preview", info.preview());
+        builder.field("snapshot_only", EsqlFunctionRegistry.isSnapshotOnly(name));
```

Comment:
`qstr` is a snapshot function, and it has a `QueryStringFunctionTests`, it seems like it is not marked as snapshot_only in the doc somehow. `def(QueryStringFunction.class, QueryStringFunction::new, "qstr")`

Reply:
Thanks for checking @fang-xing-esql 
I ran `QueryStringFunctionTests` manually from IntelliJ and now I see it flagged correctly.
I'm pushing the change, but I wonder why it didn't show up initially, maybe I just forgot to push that file...